src/plugins/preauth/pkinit/pkinit_crypto_openssl.c

```diff
@@ -3707,8 +3719,8 @@ pkinit_open_session(krb5_context context,
         }
         label_len = cp - tinfo.label;
 
-        pkiDebug("open_session: slotid %d token \"%.*s\"\n",
-                 (int)slotlist[i], (int)label_len, tinfo.label);
+        TRACE_PKINIT_CLIENT_PKCS11_SLOT(context, (int) slotlist[i],
+                                        (size_t) label_len, tinfo.label);
```

Comment:
`label_len` is already `size_t`.

Reply:
Whoops, I think I meant that it should have been cast to `int` not `size_t`